README.md

```diff
@@ -69,7 +71,8 @@ Then you should find taosAdapter binary executable file in the working directory
 * Compatible with InfluxDB v1 write interface.  
   [https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/write/](https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/write/)
 * Compatible with opentsdb JSON and telnet format writing.  
-  [http://opentsdb.net/docs/build/html/api_http/put.html](http://opentsdb.net/docs/build/html/api_http/put.html)
+  [http://opentsdb.net/docs/build/html/api_http/put.html](http://opentsdb.net/docs/build/html/api_http/put.html)  
```

Comment:
Recommanded bare url style in Markdown for GitHub:

```markdown
<http://opentsdb.net/docs/build/html/api_http/put.html>
```

Rule: don't repeat yourself.